reporting/src/test/scala/caliban/reporting/MockSchemaReporter.scala

```diff
@@ -0,0 +1,20 @@
+package caliban.reporting
+
+import zio.test.mock
```

Comment:
This module was recently removed from ZIO core, I think because it is not as well maintained as the rest. I wonder if this might bite us later. Is it hard to do without?

Reply:
Probably not that hard, it’s just a single method that needs to be mocked